frontend/packages/dev-console/src/components/import/serverless-function/AddServerlessFunctionForm.tsx

```diff
@@ -147,10 +147,22 @@ const AddServerlessFunctionForm: React.FC<
                   title={t('devconsole~func.yaml is not present and builder strategy is not s2i')}
                 >
                   <p>
-                    {t(
-                      'devconsole~func.yaml must be present and builder strategy should be s2i to create a Serverless function',
-                    )}
+                    <Trans t={t} ns="devconsole">
+                      The Function cannot be built from this repository since it's not created with{' '}
+                      <code className="co-code">kn func create</code> command.
+                    </Trans>
                   </p>
+                  <a
+                    href={
+                      'https://docs.openshift.com/serverless/1.31/functions/serverless-functions-getting-started.html'
+                    }
```

Comment:
What will happen if version gets changed? and what if we backport this to 4.13 or 4.12, there serverless 1.31 is supported? and if 1.32 is released do we need to update the url?